pyproject.toml

```diff
@@ -17,7 +17,7 @@ python = "^3.7"
 certifi = "==2018.10.15"
 chardet = "==3.0.4"
 idna = "==2.7"
-lxml = "==4.5.1"
+lxml = "^4.6.3"
```

Comment:
PR #250 increased the version of `lxml` to `^4.9.1`. So this change would be a `lxml` version downgrade - feel free to bump the `lxml` version to a more recent version